### PR TITLE
#433 Added checks on environment variables if they are not defined.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/sources/API/CRV/CRVXmlImporter.cpp [hunk 1]
```diff
@@ -57,6 +57,8 @@ XmlImporter::importFromStream(std::istream& stream) const {
   try {
     bool xsdValidation = false;
     if (getEnvVar("DYNAWO_USE_XSD_VALIDATION") == "true") {
+      if (!hasEnvVar("DYNAWO_XSD_DIR"))
```
Comment (rosiereflo) [inline dynawo/sources/API/CRV/CRVXmlImporter.cpp]: Why not factorizing the test inside getEnvVar method?

File: dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic [hunk 2]
```diff
@@ -185,6 +185,7 @@ AutomatonMaximumOutputSizeReached =       automaton %1% has more outputs defined
 SystemCallFailed            =             call to %1% failed with error '%2%'
 MultipleErrors              =             %1% errors were found and displayed above
 MultipleAndHiddenErrors     =             first %1% errors displayed above
+MissingEnvironmentVariable  =             environment variable %1% is missing, you need to define it to run Dynawo
```
Comment (rosiereflo) [inline dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: I would propose "mandatory environment variable %1% is not defined"

Comment (gautierbureau) [reply dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: ok done

File: dynawo/sources/API/CRV/CRVXmlImporter.cpp (hunk 1 above)
Comment (gautierbureau) [reply dynawo/sources/API/CRV/CRVXmlImporter.cpp]: If we do this all environment variables become mandatory and they are not at the moment. Or we could have to methods getEnvVar and getMandatoryEnvVar I guess

Comment (rosiereflo) [reply dynawo/sources/API/CRV/CRVXmlImporter.cpp]: I agree with this proposal

Comment (gautierbureau) [reply dynawo/sources/API/CRV/CRVXmlImporter.cpp]: ok i'll do that

**Pull request merged**